app/services/finance/npq/statement_calculator.rb

```diff
@@ -47,30 +47,21 @@ def total_payment
       end
 
       def total_starts
-        statement
-          .billable_statement_line_items
-          .joins(:participant_declaration)
-          .where(participant_declarations: { declaration_type: "started" })
-          .merge(ParticipantDeclaration.select("DISTINCT (user_id, declaration_type)"))
-          .count
+        contracts.sum do |contract|
+          CourseStatementCalculator.new(statement:, contract:).billable_declarations_count_for_declaration_type("started")
```

Comment:
also seem we'd do just fine with a single instance of the `CourseStatementCalculator` 
just pass the contract as an argument to the methods. Would remove the need to instantiate a bunch of objects per contract / per declaration type

Reply:
after checking the class it appears we don't even use the contract. Can you ✂️  that please.

Reply:
not sure I follow:
- `CourseStatementCalculator` does use the `contract` that is passed so I can't remove that. Just because the method doesn't use it does not mean i can remove it from initialisation. 
- There are multiple courses so I expect `CourseStatementCalculator` to be instantiated multiple times, not once